app/assets/stylesheets/library/_colors.scss

```diff
@@ -1,9 +1,8 @@
-$dark-green: #357c66;
 $dark-grey: #333;
 $offwhite: #f2f2f2;
-$light-blue: #DAE8ED;
-$blue: #B8D4DE;
 $grey: #d0d0d0;
 $light-grey: #f2f2f2;
-$light-green: #c2d6d6;
-$water-green: #e8fff8;
+$pale-green: #8caba9;
+$pale-lily: #d7c6cf;
+$pale-pink: #ebdada;
+$pale-prune: #a2798f;
```

Comment:
Nice use of variables here and below!

Reply:
I just suggested you to do the same in yours 😄 I found it very useful and easy to change!

Reply:
As above, can you alphabetise these?

Reply:
👍  done